rosdep/python.yaml

```diff
@@ -5579,6 +5579,11 @@ python3-bitarray:
   fedora: [python3-bitarray]
   gentoo: [dev-python/bitarray]
   ubuntu: [python3-bitarray]
+python3-bluez:
+  debian: [python3-bluez]
+  fedora: [python3-bluez]
+  gentoo: [dev-python/pybluez]
+  ubuntu: [python3-bluez]
```

Comment:
It looks like `python3-bluez` is only supported on `focal` forward, so I'll suggest:

```
  ubuntu:
    '*': [python3-bluez]
    'bionic': null
    'xenial': null
```

Reply:
Good point, added as a suggestion in https://github.com/ros/rosdistro/pull/28495#discussion_r589637724